pkg/daemon/ceph/client/pool.go

```diff
@@ -233,9 +234,8 @@ func GetErasureCodeProfileForPool(baseName string) string {
 	return fmt.Sprintf("%s_ecprofile", baseName)
 }
 
-func CreateECPoolForApp(context *clusterd.Context, namespace, poolName, ecProfileName string, pool cephv1.PoolSpec, appName string, enableECOverwrite bool) error {
-	args := []string{"osd", "pool", "create", poolName, "0", "erasure", ecProfileName}
-
+func CreateECPoolForApp(context *clusterd.Context, namespace, poolName, ecProfileName string, pool cephv1.PoolSpec, pgCount, appName string, enableECOverwrite bool) error {
```

Comment:
```suggestion
func CreateECPoolForApp(context *clusterd.Context, namespace, poolName, ecProfileName, pgCount, appName string, pool cephv1.PoolSpec, enableECOverwrite bool) error {
```

Reply:
i'd rather not reorder the params for this change